de_templates/ingest/Ingest_AzureSql_SourceName/data_factory/pipelines/Ingest_AzureSql_SourceName.json.jinja

```diff
@@ -0,0 +1,237 @@
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "factoryName": {
+            "type": "string",
+            "metadata": "Data Factory name"
+        }
+    },
+    "variables": {
+        "factoryId": "[concat('Microsoft.DataFactory/factories/', parameters('factoryName'))]"
+    },
+    "resources": [
+        {
+            "name": "[concat(parameters('factoryName'), '/Ingest_AzureSql_{{ source_name }}')]",
+            "type": "Microsoft.DataFactory/factories/pipelines",
+            "apiVersion": "2018-06-01",
+            "properties": {
+                "description": "Ingest from demo Azure SQL database using ingest config file.",
+                "activities": [
+                    {
+                        "name": "Get_Ingest_Config",
+                        "type": "ExecutePipeline",
+                        "dependsOn": [],
+                        "userProperties": [],
+                        "typeProperties": {
+                            "pipeline": {
+                                "referenceName": "Get_Ingest_Config",
+                                "type": "PipelineReference"
+                            },
+                            "waitOnCompletion": true,
+                            "parameters": {
+                                "config_container": "config",
+                                "config_path": {
+                                    "value": "ingest/@{pipeline().parameters.data_source_name}/ingest_sources",
+                                    "type": "Expression"
+                                },
+                                "config_file": {
+                                    "value": "@{pipeline().parameters.data_source_name}.json",
+                                    "type": "Expression"
+                                }
+                            }
+                        }
+                    },
+                    {
+                        "name": "For_Each_Ingest_Entity",
+                        "type": "ForEach",
+                        "dependsOn": [
+                            {
+                                "activity": "Get_Ingest_Config",
+                                "dependencyConditions": [
+                                    "Succeeded"
+                                ]
+                            }
+                        ],
+                        "userProperties": [],
+                        "typeProperties": {
+                            "items": {
+                                "value": "@activity('Get_Ingest_Config').output.pipelineReturnValue['config'][0]['ingest_entities']",
+                                "type": "Expression"
+                            },
+                            "isSequential": false,
+                            "activities": [
+                                {
+                                    "name": "SQL_to_ADLS",
```

Comment:
Could make this pipeline name dynamic based on the data source (or, call it `Source_to_ADLS`)

Reply:
done